moesifwsgi/app_config.py

```diff
@@ -47,12 +47,20 @@ def get_sampling_percentage(cls, config, user_id, company_id):
 
             company_sample_rate = config_body.get('company_sample_rate', None)
 
+            regex_config = config_body.get('regex_config', None)
+
             if user_id and user_sample_rate and user_id in user_sample_rate:
                 return user_sample_rate[user_id]
 
             if company_id and company_sample_rate and company_id in company_sample_rate:
                 return company_sample_rate[company_id]
 
+            if regex_config:
```

Comment:
@keyur9 , I think we made the regex sampling higher priority than user behavior based sampling. See https://www.moesif.com/docs/platform/dynamic-sampling/#how-to-set-sample-rates

Reply:
okay, have refactored so that regex sampling have higher priority than entity sampling.